src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -733,6 +733,19 @@ void ObjectMgr::LoadCreatureSummonedData()
             }
         }
 
+        if (!fields[4].IsNull())
+        {
+            for (std::string_view questStr : Trinity::Tokenize(fields[4].GetStringView(), ',', false))
+            {
+                if (Optional<uint32> questId = Trinity::StringTo<uint32>(questStr))
+                {
+                    // quests aren't loaded yet, we cannot validate them
+                    summonedData.QuestsToDespawnOnQuestRemove.emplace();
```

Comment:
This should be done before the for loop (emplace destroy already existing value if any)